### PR TITLE
Disable TopK e2e test on Vulkan

### PR DESCRIPTION
**Pull request by ThomasRaoux**
Disable e2e test while investigating vulkan failure.

### CODE REVIEW

**Comment by MaheshRavishankar** [comment]
Could we just disable the e2e Vulkan test? CPU side seems fine.

**Comment by ThomasRaoux** [comment]
> Could we just disable the e2e Vulkan test? CPU side seems fine.

sure, changed it

**Review by hanhanW — APPROVED** [review]

**Review by KoolJBlack — APPROVED** [review]
I see this only reverts the vulkan change which is fine by me. 

Can I trigger the post-submit vulkan tests on a PR before post-submit to ensure my fix addresses the issue?

**Review by MaheshRavishankar — APPROVED** [review]

File: iree/test/e2e/linalg_ext_ops/BUILD [hunk 1]
```diff
@@ -72,7 +72,8 @@ iree_check_single_backend_test_suite(
         # keep sorted
         [
             "reverse.mlir",
-            "top-k.mlir",
+            # Top-k test disabled due to miscompile on vulkan.
+            #    "top-k.mlir",
```
Comment (MaheshRavishankar) [inline iree/test/e2e/linalg_ext_ops/BUILD:76]: THis is probaby going to fail presubmit if not added to the "explicit" list.

**Comment by ThomasRaoux** [comment]
> I see this only reverts the vulkan change which is fine by me.
> 
> Can I trigger the post-submit vulkan tests on a PR before post-submit to ensure my fix addresses the issue?

Yes I think there is a tag you can set to run it

File: iree/test/e2e/linalg_ext_ops/BUILD (hunk 1 above)
Comment (ThomasRaoux) [reply iree/test/e2e/linalg_ext_ops/BUILD:76]: yes it did, updated it

**Pull request merged**